single-line-diagram-core/src/main/java/com/powsybl/sld/svg/GraphMetadata.java

```diff
@@ -47,6 +47,8 @@
 
         private final boolean vLabel;
 
+        private final String unescapedId;
+
```

Comment:
this field correspond to an equipmentId : so maybe we can rename it ?

Reply:
You are right, equipmentId would be more meaningful